source/documentation/monitoring_apps/metrics.md

```diff
@@ -123,3 +123,84 @@ If you are not receiving any metrics, check the [logs](/monitoring_apps.html#log
 ### More about monitoring
 
 For more information about monitoring apps, see [Monitoring the status of your service](https://www.gov.uk/service-manual/technology/monitoring-the-status-of-your-service) on the Service Manual.
+
+## Backing service metrics
+
+If you use the [PostgreSQL](deploying_services/postgresql/#postgresql), [MySQL](deploying_services/mysql/#mysql) or [Redis](deploying_services/redis/#redis) backing service, you can view metrics on those backing services in the GOV.UK PaaS admin tool.
```

Comment:
> If you use ~the~ a [PostgreSQL](deploying_services/postgresql/#postgresql), [MySQL](deploying_services/mysql/#mysql) or [Redis](deploying_services/redis/#redis) backing service, you can view metrics on ~those backing services~ it in the GOV.UK PaaS admin tool.

Thoughts?

Reply:
Our style is that we try to minimise the use of "it" to avoid ambiguity, especially when there are several things "it" could refer to. Means the text is a bit clunky sometimes, but it is explicitly clear.